mq/plugins/broFixup.py

```diff
@@ -518,17 +518,24 @@ def onMessage(self, message, metadata):
                         newmessage['details']['client'] = 'unknown'
                     if 'service' not in newmessage['details']:
                         newmessage['details']['service'] = 'unknown'
-                    if 'success' not in newmessage['details']:
-                        newmessage['details']['success'] = 'unknown'
                     if 'error_msg' not in newmessage['details']:
                         newmessage['details']['error_msg'] = ''
-                    newmessage['summary'] = (
-                        '{sourceipaddress} -> '
-                        '{destinationipaddress}:'
-                        '{destinationport} '
-                        'request {request_type} '
-                        'success {success}'
-                    ).format(**newmessage['details'])
+                    if 'success' not in newmessage['details']:
```

Comment:
This can be cleaned up a bit:
```
newmessage['summary'] = (
    '{sourceipaddress} -> '
    '{destinationipaddress}:'
    '{destinationport} '
    'request {request_type} '
).format(**newmessage['details'])
if 'success' in newmessage['details']:
    newmessage['summary'] += 'success {0}'.format(newmessage['details']['success'])
else:
    newmessage['summary'] += 'success unknown'
```

Reply:
added, thanks for the suggestion!